torchvision/prototype/features/_feature.py

```diff
@@ -16,13 +16,14 @@ def __new__(
         device: Optional[Union[torch.device, str, int]] = None,
         requires_grad: bool = False,
     ) -> F:
-        return cast(
-            F,
-            torch.Tensor._make_subclass(
-                cast(_TensorBase, cls),
-                torch.as_tensor(data, dtype=dtype, device=device),  # type: ignore[arg-type]
-                requires_grad,
-            ),
+        return (
+            torch.as_tensor(  # type: ignore[return-value]
+                data,
+                dtype=dtype,
+                device=device,  # type: ignore[arg-type]
+            )
+            .as_subclass(cls)  # type: ignore[arg-type]
```

Comment:
@NicolasHug While looking into the dispatch, I found that this is recommended and documented way to create tensor subclasses. I recall, you mentioned we should look into finding such a solution when we went over the proposal.